plugins/ProfileExtender/class.profileextender.plugin.php

```diff
@@ -236,6 +236,8 @@ private function profileFields($Sender) {
 
         // Get user-specific data
         $this->UserFields = Gdn::userModel()->getMeta($Sender->Form->getValue('UserID'), 'Profile.%', 'Profile.');
+        $this->EventArguments['userFields'] = &$this->UserFields ?? '';
```

Comment:
You could use the $sender object instead of passing an event argument by reference, objects are always passed by reference as opposed to arrays.